src/Sylius/Bundle/AdminBundle/Resources/config/app/events.yaml

```diff
@@ -192,8 +164,6 @@ sylius_ui:
                 legacy:
                     template: "@SyliusUi/Block/_legacySonataEvent.html.twig"
                     priority: 0
-                    context:
-                        event: sylius.admin.layout.stylesheets
```

Comment:
Why are you removing all events? Are they not needed for legacy sonata to generate proper event names?

Reply:
Right, sometimes they are postfixed so I use `sonata_postfix` in the context in such cases.

Reply:
@GSadee As you noticed there are much more sonata events with a different name than the template block names. Therefore I've deleted `LegacySonataEventContextProvider` completely.